views/layout.erb

```diff
@@ -86,7 +66,7 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="/"><img src="/img/logo/slovenskodigital-logo.svg" alt="Slovensko digital"></a>
+      <a class="navbar-brand" href="/"><img src="img/logo/slovenskodigital-logo.svg" alt="Slovensko digital"></a>
```

Comment:
Ak vyhodis tieto rooty tak sa to pravdepodobne rozbije na sublinkach, nie je to tam nahodou.